pkg/recorder/chrome.go

```
@@ -0,0 +1,129 @@
+		r.closeChrome()
+		r.closeChrome = nil
+	}
+}
+
+func (r *Recorder) waitVisibleWithTimeout(selector string, timeout time.Duration) chromedp.ActionFunc {
+	return func(ctx context.Context) error {
+		timeoutCtx, cancel := context.WithTimeout(ctx, timeout)
+		defer cancel()
+		err := chromedp.WaitVisible(selector).Do(timeoutCtx)
+		if err != nil && errors.Is(err, context.DeadlineExceeded) {
+			err = errors.New(fmt.Sprintf("%s was not visible after %v", selector, timeout))
```

golangci-lint (1.62.2)

S1028: should use fmt.Errorf(...) instead of errors.New(fmt.Sprintf(...))

(gosimple)